spec/controllers/base_media_controller_spec.rb

```diff
@@ -3,7 +3,7 @@
 RSpec.describe BaseMediaController, type: :controller do
   controller do
     def anything
-      render nothing: true
+      head :ok
```

Comment:
I think this is related to the previous commit, isn't it?

Reply:
You're correct! I've put it in the commit before. Missed it whilst splitting out